docs/user/commercial/privacy-level.rst

```diff
@@ -1,20 +1,41 @@
-.. TODO: This is a super weird page..
+Privacy Levels
+--------------
 
-:orphan:
+.. include:: /shared/admonition-rtd-business.rst
 
-Project privacy level
----------------------
+Privacy levels allow you to set the visibility of your project and its versions.
+This allows you to specify what information is available to the public and what is private.
+Privacy is generally controlled at the level of the version,
+and nothing more granular (eg. specific directories or URLs) can be controlled.
 
-.. include:: /shared/admonition-rtd-business.rst
+Project privacy
+~~~~~~~~~~~~~~~
 
 By default, only users that belong to your organization can see the dashboard of your project and its builds.
-If you want users outside your organization and anonymous users to be able to see the dashboard of your project,
+If you want users outside your organization to able to see the dashboard of your project,
 and the build output of *public versions* you can set the privacy level of your project to ``Public``.
```

Comment:
Maybe makes sense to explicitly say here this doesn't give access to the documentation. That will make this behavior clearer.